build-data/generated/tracking_parameters.json

```diff
@@ -0,0 +1,902 @@
+                    "id.a-mx.com": 0.074,
+                    "Sailthru, Inc": 0.064,
+                    "Oracle Corporation": 0.061,
+                    "ANIVIEW LTD": 0.048,
+                    "Akamai Technologies": 0.029,
+                    "Index Exchange, Inc.": 0.028,
+                    "Salesforce.com, Inc.": 0.024,
+                    "Smartadserver S.A.S": 0.019
+                }
+            },
+            "cookies": {
+                "prevalence": 0,
+                "firstParty": 17,
+                "thirdParty": 42,
+                "entities": {
+                    "mcknightsseniorliving.com": 0.167,
+                    "Haymarket Media Group Ltd.": 0.167,
+                    "renalandurologynews.com": 0.167,
+                    "concealedcarry.com": 0.095,
+                    "audacy.com": 0.071,
+                    "The Nasdaq Stock Market LLC": 0.048,
+                    "Auth0, Inc.": 0.048,
+                    "urban-vpn.com": 0.048,
+                    "expresso.pt": 0.048
+                }
+            }
+        },
+        "action_ref_map": {
+            "prevalence": 0.078,
+            "exampleSites": [
+                "deals.dell.com/pt-br",
+                "www.usnews.com/topics/subjects/jobs-reports",
+                "reg.usps.com/restoreaccess",
+                "www.verywellhealth.com",
+                "www.tasteofhome.com",
+                "es.digitaltrends.com/videojuego/juegos-multiplataforma",
+                "creditcards.usnews.com/zero-interest",
+                "prezi.com",
+                "forums.commentcamarche.net/forum/affich-8418702-nikon-coolpix-s500-pb-bloc-optique",
+                "globalnews.ca"
+            ],
+            "requests3p": {
+                "prevalence": 0.078,
+                "entities": {
+                    "Google LLC": 0.144,
+                    "id.a-mx.com": 0.073,
+                    "Sailthru, Inc": 0.064,
+                    "ANIVIEW LTD": 0.046,
+                    "Bazaarvoice, Inc.": 0.039,
+                    "Oracle Corporation": 0.038,
+                    "Akamai Technologies": 0.028,
+                    "Index Exchange, Inc.": 0.027,
+                    "Salesforce.com, Inc.": 0.024
+                }
+            },
+            "cookies": {
+                "prevalence": 0,
+                "firstParty": 17,
+                "thirdParty": 40,
+                "entities": {
+                    "mcknightsseniorliving.com": 0.175,
+                    "Haymarket Media Group Ltd.": 0.175,
+                    "renalandurologynews.com": 0.175,
+                    "concealedcarry.com": 0.1,
+                    "The Nasdaq Stock Market LLC": 0.05,
+                    "Auth0, Inc.": 0.05,
+                    "urban-vpn.com": 0.05,
+                    "expresso.pt": 0.05,
+                    "lawofselfdefense.com": 0.05
+                }
+            }
+        },
+        "fb_source": {
+            "prevalence": 0.096,
```

Comment:
it *almost* seemsl like you sorted params by prevalence, but suddenly this? Is that expected?

Reply:
It's not sorted